example.py

```diff
@@ -148,37 +128,5 @@ def main():
     conn.close()
 
 
-def parse_cmdline():
-    parser = ArgumentParser(description=__doc__,
-                            formatter_class=RawTextHelpFormatter)
-    parser.add_argument(
-        "dsn",
-        help="""\
-database connection string
```

Comment:
Again, lacking context, but are you doing a favour to your user by removing all this documentation?